src/Connection.cpp

```diff
@@ -484,10 +489,23 @@ std::unique_ptr<sdbus::IConnection> createConnection(const std::string& name)
     return createSystemBusConnection(name);
 }
 
+std::unique_ptr<sdbus::IConnection> createDefaultBusConnection()
+{
+    auto interface = std::make_unique<sdbus::internal::SdBus>();
+    constexpr sdbus::internal::Connection::default_bus_t default_bus;
+    return std::make_unique<sdbus::internal::Connection>(std::move(interface), default_bus);
+}
+
+std::unique_ptr<sdbus::IConnection> createDefaultBusConnection(const std::string& name)
+{
+    auto conn = createConnection();
```

Comment:
Shouldn't we call `createDefaultBusConnection()` here?

Reply:
Yes, good catch. This was left over from when I had `createConnection()` creating the default bus connection.